### PR TITLE
Update suppressions.xml added temporary suppression for CVE-2022-34305

### PR DESCRIPTION
**Pull request by dinesh1patel**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-3433

### Change description ###

Update suppressions.xml added temporary suppression for CVE-2022-34305

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -23,6 +23,15 @@
     </notes>
     <cve>CVE-2022-29885</cve>
   </suppress>
+
```
Comment (rajaram-nagarajan) [inline config/owasp/suppressions.xml:26]: @dinesh1patel can you please move all the temporary suppression to one element?

Comment (dinesh1patel) [reply config/owasp/suppressions.xml]: @rajaram-nagarajan done

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -16,12 +16,11 @@
...
   <suppress>
-    <notes><![CDATA[
-      file name: tomcat-embed-core-9.0.58.jar
-      ]]>
-      https://tools.hmcts.net/jira/browse/CCD-3270
-    </notes>
+    <notes>
+      CVE-2022-29885 refer https://tools.hmcts.net/jira/browse/CCD-3270
+      CVE-2022-34305 refer https://tools.hmcts.net/jira/browse/CCD-3433
     <cve>CVE-2022-29885</cve>
```
Comment (rajaram-nagarajan) [inline config/owasp/suppressions.xml:22]: @dinesh1patel - notes end tag seems to be missing

Comment (dinesh1patel) [reply config/owasp/suppressions.xml:23]: @rajaram-nagarajan done

**Pull request merged**